pkg/app/app.go

```diff
@@ -109,12 +109,6 @@ func (c *App) Handler() http.Handler {
 	// Host Orchestrator Proxy Routes
 	router.Handle("/v1/zones/{zone}/hosts/{host}/{hostPath:.*}", c.Authenticate(c.ForwardToHost))
 
-	// Infra route
-	router.HandleFunc("/v1/zones/{zone}/hosts/{host}/infra_config", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
let's move the route declaration on top of `router.Handle("/v1/zones/{zone}/hosts/{host}/{hostPath:.*}", c.Authenticate(c.ForwardToHost))`, this way it would get handled first. 

To make sure it's always handled at the CO, let's add a unit test for it.

Reply:
> To make sure it's always handled at the CO, let's add a unit test for it.

How existing unit [tests](https://github.com/google/cloud-android-orchestration/blob/main/pkg/app/instances/gce_test.go) are managed? How I can rerun those tests? And I'm also curious if there's an existing CI in public repository or internally.

Reply:
> let's move the route declaration on top of router.Handle("/v1/zones/{zone}/hosts/{host}/{hostPath:.*}", c.Authenticate(c.ForwardToHost)), this way it would get handled first.

Done for this.

Reply:
Unit tests are found in [app_test.go](https://github.com/google/cloud-android-orchestration/blob/main/pkg/app/app_test.go) you can use `TestListZonesSucceeds` as reference.